src/heist_path/HeistPathB.py

```
@@ -484,16 +503,85 @@ def lg2_wp2_to_exit2_safer(self):
+    def switch_to_fighter(self, check_switched=False, mode="all_desc"):
+        """切换到可用战斗角色。   
+        `mode` 调度策略（配置重新从小到大排序后）：
+        - "all_desc": [默认]按键位从大到小完整尝试（如 ["4", "1"]）
+        - "all_asc" : 按键位从小到大完整尝试（如 ["1", "4"]）
+        -     1     : 只切当前最小的那个键位
+        -    -1     : 只切当前最大的那个键位
+        -     n     : 只切重新排序后的【第 n 个】角色
+        """
+        config_keys = list(self.config.get(self.CONF_FIGHTER, []))
+        if not config_keys:
+            self.log_error("配置中的战斗角色列表为空！")
```

Ruff (0.15.14)

[warning] String contains ambiguous `！` (FULLWIDTH EXCLAMATION MARK). Did you mean `!` (EXCLAMATION MARK)?

(RUF001)